packages/story-editor/src/components/form/color/colorInput.js

```diff
@@ -242,9 +248,19 @@ const ColorInput = forwardRef(function ColorInput(
                 </Text>
               </TextualPreview>
             ) : (
-              <ChevronContainer>
-                <Icons.ChevronDown />
-              </ChevronContainer>
+              <>
+                {/* In case of design menu, we display Mixed value even without inputs */}
+                {isInDesignMenu && isMixed && (
```

Comment:
@barklund Per UX, we should display `Mixed` in the menu if the text includes more than one color, and it should look similar to the Design Panel.

PTAL at these changes here -- feels a bit hacky but not sure if there's a better way.

Reply:
The chevron is located wrong now. It should be vertically centered, but is too high now (both with and without the mixed label):

| Before | After |
|-|-|
| ![Screen Shot 2022-03-08 at 15 22 06](https://user-images.githubusercontent.com/637548/157318868-412278bc-0247-4abd-a7e5-61dab9bdd587.png) | ![Screen Shot 2022-03-08 at 15 24 15](https://user-images.githubusercontent.com/637548/157318919-d9c59ec0-dfe3-42e3-a8b2-f9641ff5eee7.png) |


As for the code, I think we can remove the `isInDesignMenu && ` and just say that we always show mixed even if inputs are off. Currently this would only happen in the design menu anyway, but if we were later to have no-input color pickers elsewhere, the same thing should probably happen.

Reply:
Makes sense, done!